Sofascore2025/Sofascore2025/View/HeaderView.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+protocol HeaderViewDelegate: AnyObject {
+    func didTapSettings()
+}
+
+class HeaderView: BaseView {
+    private let logoImageView = UIImageView()
+    private let tropheyImageView = UIImageView()
+    private let settingsImageView = UIImageView()
+    private let settingsContainerView = UIView()
+    private let tropheyContainerView = UIView()
+    
+    weak var delegate: HeaderViewDelegate?
+
+    override func addViews() {
+        addSubview(logoImageView)
+        addSubview(tropheyImageView)
+        addSubview(settingsContainerView)
+        addSubview(tropheyContainerView)
+        
+        settingsContainerView.addSubview(settingsImageView)
+        tropheyContainerView.addSubview(tropheyImageView)
+        
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(didTapSettingsIcon))
+        settingsImageView.isUserInteractionEnabled = true
+        settingsImageView.addGestureRecognizer(tapGesture)
+    }
+
+    @objc private func didTapSettingsIcon() {
+        delegate?.didTapSettings()
+    }
+    
+    override func styleViews() {
+        backgroundColor = .customBlue
+        
+        logoImageView.image = UIImage(named: "sofascore_lockup")
+        tropheyImageView.image = UIImage(named: "Icon 2")
+        settingsImageView.image = UIImage(named: "Icon 1")
+    }
+
+    override func setupConstraints() {
+        logoImageView.snp.makeConstraints {
+            $0.leading.equalToSuperview().offset(16)
+            $0.top.equalToSuperview().offset(14)
```

Comment:
Inset koristiti u slucaju superview-a dok offset kad je na neki drugi view
```suggestion
            $0.top.equalToSuperview().inset(14)
```